src/pages/studyView/studyPageHeader/ActionButtons.tsx

```diff
@@ -20,7 +20,7 @@ export interface ActionButtonsProps {
     appStore: AppStore;
 }
 
-export const MAXIMUM_NAV_CASE_IDS_IN_URL = 2000;
+export const MAXIMUM_NAV_CASE_IDS_IN_URL = 100;
```

Comment:
can prolly remove this variable now that we use `4500` characters limit directly for the url length check